source/19-Psalms.usfm.db

```diff
@@ -2988,7 +2988,7 @@
 \v 8 for the ransom of a life is costly,
 \q2 no payment is ever enough,
 \q
-\v 9 to keep [neut:them|masc:him] alive for ever and ever,
+\v 9 to keep [neut:them|masc:him] alive [us:forever|cth:for ever] and ever,
```

Comment:
I think the "for ever" here should remain split given the fact that it is part of the phrase "for ever and ever".